src/Utils.vala

```diff
@@ -61,4 +61,22 @@ public class Mail.Utils {
     public static string escape_html_tags (string input) {
         return input.replace ("<", "&lt;").replace (">", "&gt;");
     }
+
+    public static string strip_html_tags (string input) {
+        try {
+            var regex = new Regex ("<[^>]*>");
+            return regex.replace (input, input.length, 0, "");
+
+        } catch (RegexError e) {
+            warning ("Error compiling regex to strip html tags: %s", e.message);
+            return input;
+        }
+    }
+
+    public static string? strip_folder_full_name (string service_uid, string? folder_uri) {
+        if (folder_uri != null) {
+            return folder_uri.replace ("folder://%s/".printf (service_uid), "");
```

Comment:
I wonder if we also need to unescape here. I get an error about my Gmail drafts folder since the path is `[Gmail]/Drafts`:

`Unable to safe draft: No such folder %5bGmail%5d/Drafts`

Reply:
Added Camel.URL.decode when retrieving the drafts folder.